library/scd4x/__init__.py

```diff
@@ -1,8 +1,9 @@
-import time
 import struct
+import time
+
 from smbus2 import SMBus, i2c_msg
 
-__version__ = '0.0.1'
+__version__ = "0.0.1"
```

Comment:
I believe this change will break the version checks in my very tenuous Makefile. Though the answer might be to fix the Makefile with something a bit less fragile.

Reply:
Seems less fragile than you might think :)

```
pi@raspberrypi:~/git/scd4x-python $ grep version library/scd4x/__init__.py 
__version__ = "0.0.1"
pi@raspberrypi:~/git/scd4x-python $ make usage
Library: scd4x
Version: 0.0.1
(...)
```